ecl2df/common.py

```diff
@@ -293,14 +293,21 @@ def parse_opmio_deckrecord(
                     rec_dict[item_name] = record[item_idx].get_data_list()
                 # (the caller is responsible for unrolling this list with
                 # correct naming of elements)
+
+                # When we parse a list, some values in it can be defaulted, for
+                # which the default values are not provided in the JSON. Return
+                # these defaulted values as NaN's for the calling code to fix.
+                for idx in range(len(record[item_idx])):
```

Comment:
Use enumerate?

Reply:
Would like to, but I get: `TypeError: 'opm.libopmcommon_python.DeckItem' object is not iterable`